style.css

```diff
@@ -1,3 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
 * {
   box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3c3a39;
+  width: 100%;
+}
+
+.logo {
+  color: #fff;
+  margin: 0 0 0 0.8em;
+  font-weight: 900;
+  cursor: pointer;
+}
+
+.ham-manu {
+  cursor: pointer;
+}
+
+.hero-sec {
+  width: 100%;
+  height: 30em;
+  background-color: #1c1a19;
+  color: #fff;
+}
```

Comment:
[Optional] Kyndly consider setting the height of the section lije this ` height: 100vh ` this will make your section take the whole viewport height of the device.

Using vh 100%
![image](https://user-images.githubusercontent.com/95243996/191747708-c016ff56-09a9-4098-929b-1d81d2869667.png)

Not using vh
![image](https://user-images.githubusercontent.com/95243996/191747861-32a2c478-4484-4d04-9110-6c282fa6d6ca.png)